task1/Program.cs

```diff
@@ -45,5 +71,792 @@ public static decimal PerformCalculation(int r)
+            x = y - x++ * z;
+            Console.WriteLine(x);
+        }
+
+        public static void Calculator()
+        {
+            int operand1 = 0;
+            int operand2 = 0;
+            string sign = "";
+            bool success = false;
+            //bool isInputCorrect = false;
+            do
+            {
+                //try
+                //{
+                //    Console.WriteLine("Input first number");
+                //    operand1 = Convert.ToInt32(Console.ReadLine());
+                //    Console.WriteLine("Input second number");
+                //    operand2 = Convert.ToInt32(Console.ReadLine());
+
+                //}
+                //catch
+                //{
+                //    isInputCorrect = true;
+                //    continue;
+                //}
+                //sign = Console.ReadLine();
+                //if (sign != "+" && sign != "-" && sign != "*" && sign != "/")
+                //{ isInputCorrect = true; }
+                //else isInputCorrect = false;
+
+                Console.WriteLine("Input first number");
+                success = Int32.TryParse(Console.ReadLine(), out operand1);
+                if (success == false)
+                    continue;
+                Console.WriteLine("Input second number");
+                success = Int32.TryParse(Console.ReadLine(), out operand2);
+                if (success == false)
+                    continue;
+                Console.WriteLine("Choose operation +, -, *, /");
+                sign = Console.ReadLine();
+                if (sign != "+" && sign != "-" && sign != "*" && sign != "/")
+                    success = false;
+
+
+
+            } while (success == false);
+
+            int result = 0;
+
+            switch (sign)
+            {
+                case "+":
+                    result = operand1 + operand2;
+                    break;
+                case "-":
+                    result = operand1 - operand2;
+                    break;
+                case "*":
+                    result = operand1 * operand2;
+                    break;
+                case "/":
+                    if (operand2 == 0)
+                    {
+                        Console.WriteLine("Can't devide by zero");
+                        return;
+                    }
+                    else
+                    { result = operand1 / operand2; }
+                    break;
+                default:
+                    break;
+            }
+            Console.WriteLine($"{operand1} {sign} {operand2} = {result}");
+        }
+
+        public static void GetNumberArea()
+        {
+            Console.WriteLine("Input number from 0 to 100");
+            int number = Convert.ToInt32(Console.ReadLine());
+            if (number >= 0 && number <= 14)
+            {
+                Console.WriteLine($"{number} is in area [0-14]");
+            }
+            if (number >= 15 && number <= 35)
```

Comment:
I think we should skip further conditions if one of previous was succeeded.
So we need to write else if statement.